01-CodeReviewEx/Shop-Ex/Program.cs

```diff
@@ -12,9 +12,33 @@ static void Main(string[] args)
         {
             
          
+         Account[] myAccounts = new Account[10];
+            Account a = new Account();
+            a.principal = 1.0;
+            a.rate = 0.5;
+            myAccounts[0] = a;
+            a = new Account() { principal = 1.5, rate = 0.75 };
+            
+            double fee = calculateFee(myAccounts);
+            Console.WriteLine(fee);
+         
+        }
+
+        public static double calculateFee(Account[] accounts)
+        {
+            double totalFee = 0.0;
+            Account account;
+            for (int i = 0; i < accounts.Length; i++)
+            {
+                account = accounts[i];
+                if (account.accountType == Account.PREMIUM || account.accountType == Account.PREMIUM_PLUS) totalFee += .0125 * ( // 1.25% broker's fee
```

Comment:
I would declare the broker's fee as a const upfront (and then reference the const's name in the function) instead of using it as a magic number with the explanation in comment